src/host/vsts.ts

```diff
@@ -8,7 +8,7 @@ export default class Vsts implements Host {
      * @example https://my-tenant.visualstudio.com/DefaultCollection/MyCollection/_git/my-repo
      * @example ssh://my-tenant@my-tenant.visualstudio.com:22/DefaultCollection/MyCollection/_git/my-repo
      */
-    private static urlRegex: RegExp = /(?:https:\/\/|ssh:\/\/.+@)(\w+)\.visualstudio\.com(?:\:\d+)?\/([/\w]+)\/_git\/([^/]+)/i;
+    private static urlRegex: RegExp = /(?:https:\/\/|ssh:\/\/.+@)(\w+)\.visualstudio\.com(?:\:\d+)?\/(.+)\/_git\/([^/]+)/i;
```

Comment:
Thank you for catching this! I am open the tenant name restriction to `.+`.

I personally could like to avoid `.+` because there may be some malicious URL like this:

```
https://i.am.bad.url/?whatever-to-match-regex=my-tenant.visualstudio.com/DefaultCollection/MyCollection/_git/my-repo
```

Then, we are leading the user to a bad URL. It is corner case, but please keep an eye on it.